experimental/ksandbox/dev/images/protoc/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM golang:1.23.5
+
+RUN apt-get update; apt-get install --yes unzip
+
+RUN go install google.golang.org/protobuf/cmd/protoc-gen-go@v1.28
+RUN go install google.golang.org/grpc/cmd/protoc-gen-go-grpc@v1.2
+
+# TODO: Compile ourselves?
+RUN mkdir -p /opt/protoc
+RUN wget -N -O /opt/protoc.zip https://github.com/protocolbuffers/protobuf/releases/download/v3.12.4/protoc-3.12.4-linux-x86_64.zip
```

Comment:
We make the assumption we run locally this on x86 CPUs and exclude ARM environments. Maybe add more parameters to handle at least `{linux,darwin}` ?
_Happy to do it as a follow-up PR_

Reply:
Yes, that's a good call.  Maybe compiling it ourselves is a good option from that perspective also!  (It should get cached after the first `docker buildx`)